wagtail/admin/templates/wagtailadmin/pages/confirm_delete.html

```diff
@@ -68,6 +68,20 @@
 
         <form action="{% url 'wagtailadmin_pages:delete' page.id %}" method="POST">
             {% csrf_token %}
+            {% if confirm_before_delete %}
+                <p class="status-msg warning">
+                    {% blocktrans with total_pages=descendant_count|add:1 %}
+                        This action will delete total <b>{{total_pages}}</b> pages.
+                    {% endblocktrans %}
+                </p>
+                <p>
+                    {% blocktrans %}
+                        Please type <b>{{wagtail_site_name}}</b> to confirm.
+                    {% endblocktrans %}
+                    <input type="text" name="confirm_site_name" required>
+                </p>
+                <br>
```

Comment:
best to avoid additional break elements if possible, instead add a utility class to the `p` if required `w-mb-4` or `w-mb-8` should do it.